product_pricelist_access/models/product_pricelist.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2025 Manuel Regidor <manuel.regidor@sygel.es>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class ProductPricelist(models.Model):
+    _inherit = "product.pricelist"
+
+    access_user_ids = fields.Many2many(
```

Comment:
why don't use rols o groups? something similar to odoo chat groups could have been done.

Reply:
We need to use single users.